packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -197,7 +201,7 @@ library LibProving {
 
             if (
                 !LibBytesUtils.equal(
-                    LibBytesUtils.slice(evidence.proof, 32, 32),
+                    LibBytesUtils.slice(evidence.proofs, 34, 32),
```

Comment:
This slice operation also assumes some knowledge of the `proofs` structure. We should move it to the Verifier.